Wrapping/Generators/Python/itk/support/extras.py

```diff
@@ -837,9 +840,62 @@ def image_from_dict(image_dict: Dict) -> "itkt.Image":
     image.SetDirection(image_dict['direction'])
     return image
 
+def mesh_from_dict(mesh_dict: Dict) -> "itkt.Mesh":
+    """Deserialize an dictionary representing an itk.Mesh object."""
+    import itk
+
+    mesh = itk.Mesh.D3.New()
 
-# return an image
+    points = mesh_dict['points']
+    points = itk.vector_container_from_array(points)
+    mesh.SetPointsUsingArray(points)
 
+    cells = mesh_dict['cells']
+    cells = itk.vector_container_from_array(cells)
+    mesh.SetCellsUsingArray(cells)
+
+    return mesh
+
+def dict_from_mesh(mesh: "itkt.Mesh") -> Dict:
+    """Serialize a Python itk.Mesh object to a pickable Python dictionary."""
+    import itk
+
+    if mesh.GetNumberOfCells() == 0:
+        cells_array = np.array([], 'uint64')
+    else:
+        cells_array  = itk.array_from_vector_container(mesh.GetCellsAsArray())
+
+    if mesh.GetNumberOfPoints() == 0:
+        points_array = np.array([], 'float64')
+    else:
+        points_array  = itk.array_from_vector_container(mesh.GetPointsAsArray())
+
+    point_data = mesh.GetPointData()
+    if point_data.Size() == 0:
+        point_data = np.array([], 'float64')
+    else:
+        point_data  = itk.array_from_vector_container(point_data)
+
+    cell_data = mesh.GetCellData()
+    if cell_data.Size() == 0:
+        cell_data = np.array([], 'float64')
+    else:
+        cell_data  = itk.array_from_vector_container(cell_data)
+
+    return dict(
```

Comment:
There are a lot of empty strings in this output dictionary. It looks like a round trip serialization/de serialization would lose a fair bit of metadata from the original input.

Reply:
@GenevieveBuckley Thank you for the comments. Yes, Matt pointed to me the wasm mesh data structure ([itk wasm](https://insightsoftwareconsortium.github.io/itk-wasm/api/Mesh.html)) and I will use the same keys here. The current code has some placeholders and will be replaced with the correct values. 

I wasn't sure about some of the metadata in [MeshType](https://insightsoftwareconsortium.github.io/itk-wasm/api/MeshType.html) ex. cellPixelComponents so didn't fill it yet.